### PR TITLE
Create moving-mountains-with-yu.md

### PR DESCRIPTION
**Pull request by FrostStarBook**

### CODE REVIEW

File: submissions/moving-mountains-with-yu.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+With the upgrades of 4844, the gas consumption has become extremely low. Moreover, if you use Katana for L3, the gas will be even lower. Thanks to Katana's fast on-chain speed, I did not use a record reading method in game design. All population increases are completed through chain interaction, which also conforms to the background setting of the game's story: "The Foolish Old Man Removes the Mountains."
+
+The gameplay involves producing offspring by clicking on houses. You can increase the number of offspring per click by consuming population upgrades or by purchasing totems (each totem consumes different populations for purchase and requires certain strategies for upgrades).
+
+The unit of production is Blobert, which represents the population generated with each click. This is a virtual world, so there's no rigid generation of people.
+
+### GitHub
+Provide the GitHub where your project is hosted. Please ensure your github repo is registered with [OnlyDust](https://app.onlydust.com/p/create) to receive awards.
+> https://app.onlydust.com/p/moving-mountains-with-yu
```
Comment (tarrencev) [inline submissions/moving-mountains-with-yu.md:14]: Please provide a github url here too

File: submissions/moving-mountains-with-yu.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Provide a list of your team members.
+> - name:Frost
+> - githubs:FrostStarBook
+> - discords:FrostStar
+
+### Submission Tracks
+Provide a list of the tracks your project is submitting to.
+> - Road to Mainnet 
+> - Blobert / Realms 
+> - New World
```
Comment (tarrencev) [inline submissions/moving-mountains-with-yu.md:34]:
```suggestion
```

Since this is set in the Realms lore, it won't qualify for the New World track

**Review by tarrencev — COMMENTED** [review]
Thanks for the submission. Looking forward to checking it out!

File: submissions/moving-mountains-with-yu.md (hunk 1 above)
Comment (FrostStarBook) [reply submissions/moving-mountains-with-yu.md]: fixed

**Pull request merged**